src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/Commands/AddAzureASAccount.cs

```diff
@@ -123,6 +173,7 @@ public void OnImport()
         {
             try
             {
+                AzureSessionInitializer.InitializeAzureSession();
```

Comment:
Do not add this  - this potentially causes problems for all Azure RM modules, because you are taking chanreg of injecting depednencies used by all modules.  If your cmdlet needs to use AzureSession, then it shoudl use the existing authentication mechanism.  If it does not, then it should not initialize it.